apache/airflow
# Add decorators for external and venv python branching operators

This PR is a follow-up of PR #33356 to add also python decorators for additional branching tasks for:
- external python
- python virtual environments
To make the feature set complete also for taskflow style DAGs.

How to test?
* See docs and in general CI is green
* Run the example_branch_operator
* Run the example_branch_python_operator_decorator
